docs/source/quickstart.rst

```diff
@@ -77,12 +89,12 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
 
         if __name__ == "__main__":
             # Log a parameter (key-value pair)
-            log_param("param1", randint(0, 100))
+            log_param("config_value", randint(0, 100))
```

Comment:
it might be nice to show the dict-based logging here as well

```
# Log a collection of parameters
run_params = {
  "optimizer": "sgd",
  "seed": 98765,
  "n_iterations": 2500,
}
mlflow.log_params(run_params)
```

Reply:
TODO